core/phases/thinking.py

```
@@ -0,0 +1,239 @@
+                "action": "tool_call",
+                "tool": tool_name,
+                "arguments": arguments,
+                "thought": response.get("content", ""),
+            }
+        
+        # 回退到文本解析
+        content = response.get("content", "")
+        return self._parse_think_fallback(content)
+    
+    def _parse_think_fallback(self, response: str) -> Dict[str, Any]:
+        """回退解析：从文本中提取工具调用信息"""
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)